src/pages/Accounts/hooks/usePrivateKeyImport.ts

```diff
@@ -21,6 +21,7 @@ export const usePrivateKeyImport = () => {
           importType: ImportType.PRIVATE_KEY,
           data: utils.strip0x(privateKey),
         });
+        await selectAccount(accountId);
```

Comment:
We should have it outside of the `importPrivateKey()` function, because it can be called multiple times in loops, for example by `useKeystoreFileImport()`:

https://github.com/ava-labs/core-extension/blob/85cbf294315230af73e94e77b8cc439049844de0/src/pages/Accounts/hooks/useKeystoreFileImport.ts#L48-L65


This will activate all accounts one after another, which will cause lots of re-renders and we'll start polling balances for all of them at the same time, which will be a lot of requests.